helm/examples/azure/sa/README.md

```diff
@@ -3,12 +3,14 @@
 ## Directory contents
 
 ### Introduction
-Persistent storage can be created manually or through Kubernetes Clusters. If created through Kubernetes the storage will be gone when Persistent Volume (PV) or Kubernetes clusters are deleted. For Azure manually created shared storage with a storage account will not depend on any Kubernetes cluster. The data generated will be persistent and re-used as long the PV/PVC names and mount points are the same.
+Persistent storage can be created manually or through Kubernetes Clusters. If created through Kubernetes the storage will be gone when the Persistent Volume (PV) or Kubernetes clusters are deleted. For Azure a manually created shared storage with a storage account will not depend on any Kubernetes cluster. The data generated will be persistent and re-used as long the PV/PVC names and mount points are the same.
 
 ### Create a share with a storage account
-If you don't have a storage account and a shared volume you can create them. If you already have them, provide the information in env-sa and ../hpcc-azurefile/values.yaml and generate a Kuernetes secret, if  you do not have one yet."
-To create a shared storage including storage account you can reference Reference: https://docs.microsoft.com/en-us/azure/aks/azure-files-volume
-We provide a set of scripts to help with the process. Before you run any script make sure to login to Azure first:
+To create a shared storage including storage account you can reference Reference: https://docs.microsoft.com/en-us/azure/aks/azure-files-volume.
+The scripts in this director (sa/) will help you create a Azure storage account if you don't already have one.
```

Comment:
The scripts in this director (sa/) will help you create an Azure storage account if you don't already have one.

Reply:
Replaced